torch/ao/quantization/_pt2e/qat_utils.py

```diff
@@ -119,9 +129,12 @@ def _fuse_conv_bn_qat(m: GraphModule) -> GraphModule:
             n = n.args[0]
 
         # Copy over metadata for all three nodes in [conv - bn - getitem]
+        # Also copy over constant args for conv
         for match_pattern_node, original_node in mr.nodes_map.items():
             if original_node.target == torch.ops.aten.convolution.default:
                 replacement_conv_node.meta = original_node.meta
+                # x, weight, bias, [stride, padding, dilation, transposed, output_padding, groups]
+                replacement_conv_node.args = replacement_conv_node.args[:3] + original_node.args[3:]
```

Comment:
does this work? I remember when I'm doing similar things the original_node is actually a node in the updated graph as well so the information has been lost

Reply:
Yes I ran into that before as well. The difference here is these args are literal constant args so they're inlined. You wouldn't be able to access node args from the conv node, however, e.g. the original conv weight or bias.